README.md

```diff
@@ -47,9 +47,11 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
    echo "---" >> cmd/clusterctl/examples/google/out/provider-components.yaml
    kustomize build vendor/sigs.k8s.io/cluster-api/config/default/ >> cmd/clusterctl/examples/google/out/provider-components.yaml
    ```
-1. Create a cluster:
+2. Create a cluster, ensuring that the locally-running clusterctl process has access to GCP apis:
 
    ```bash
+   export GOOGLE_APPLICATION_CREDENTIALS=cmd/clusterctl/examples/google/out/machine-controller-serviceaccount.json
```

Comment:
This is a neat trick. I wonder if it is worth explaining why it's useful to use this service account in clusterctl.

Reply:
I added a short statement to hint that it's better to use the same credentials that your child processes will use rather than the convenient gcloud setup of the development environment (which might fail and be hard to debug).